README.rst

```diff
@@ -53,45 +45,66 @@ Example analysis script
    # Iterate through trajectories
    for ts in u.trajectory:
        print(ag.center_of_mass())
- 
-There are a number of tutorials_ on the MDAnalysis homepage that explain
-how to conduct RMSD calculations, Alignment and many more features of MDAnalysis.
 
-Source code
-===========
 
-Source code is hosted in a git repository at
+Documentation
+=============
+
+**New users** should read the `Quickstart Guide`_.
+
+**All users** should read the `User Guide`_.
+
+**Developers** may also want to refer to the `MDAnalysis API docs`_.
 
-https://github.com/MDAnalysis/mdanalysis
+There are a number of tutorials_ on the MDAnalysis homepage that
+explain how to conduct RMSD calculations, structural alignment, and
+many more features of MDAnalysis.
 
-and is available under the GNU General Public License, version 2 (see
-the file LICENSE_).
 
-This is the top level of the master repository. It contains
+Installation and availability
+=============================
 
-1. the MDAnalysis toolkit source files in the directory ::
+The latest release can be **installed via ``pip`` or ``conda``** as
+described in the `Installation Quick Start`_.
```

Comment:
As above, do we want to push for the user guide here too?

Reply:
I'd keep it directing to the webpage. It's a pretty fundamental piece of information and I like that the web page is just focused on this one thing, without much distraction around. That might be better for new users.